services/src/main/java/org/keycloak/services/cors/DefaultCorsImpl.java

```diff
@@ -36,26 +38,9 @@
 /**
  * @author <a href="mailto:sthorger@redhat.com">Stian Thorgersen</a>
  */
-public class Cors {
-
-    private static final Logger logger = Logger.getLogger(Cors.class);
-
-    public static final long DEFAULT_MAX_AGE = TimeUnit.HOURS.toSeconds(1);
-    public static final String DEFAULT_ALLOW_METHODS = "GET, HEAD, OPTIONS";
-    public static final String DEFAULT_ALLOW_HEADERS = "Origin, Accept, X-Requested-With, Content-Type, Access-Control-Request-Method, Access-Control-Request-Headers, DPoP";
-
-    public static final String ORIGIN_HEADER = "Origin";
-    public static final String AUTHORIZATION_HEADER = "Authorization";
+public class DefaultCorsImpl implements Cors {
```

Comment:
Nitpick: I'd drop the impl and go with `DefaultCors`

Reply:
I was thinking about that too. "Impl" seemed more natural as CORS is a technology and not just object. But KC convention is different (`DefaultKeycloakSession` etc.), so let's just follow it :)